mitodl/bootcamp-ecommerce
# Added automatic payment email

#### What are the relevant tickets?
closes #11 

#### What's this PR do?
Added the functionality to send automatic payment reminders 

#### How should this be manually tested?

1.  Configure mailgun, its env variables and the `MAILGUN_RECIPIENT_OVERRIDE`
2. Configure the bootcamp admission 
3. have a bootcamp/klass/installment
4. configure the installment to have a deadline 7 days (plus few hours) in the future.
5. from the django shell run call the task `mail.tasks.async_send_reminder_payment_emails.delay()`
6. You should receive an email with a reminder of the payment for the amount of the installment

If you create a fulfilled order for an amount smaller than the amount of the installment, the email should contain the difference.

If you create a fulfilled order for an amount bigger than or equal to the amount of the installment, you should not receive an email.

If you receive the email and you re-submit the task, you should not receive additional emails.
